### DISCUSSION TITLE
ability to email a note + rx notes via mail

### DISCUSSION THREAD

**Discussion by meichthys** [TriliumNext/Trilium#5938] [Plugin Ideas]
zadam/trilium#430
zadam/trilium#2650

> 
> I miss the ability to email in notes like I could in OneNote and Evernote. Obviously the way those systems work is out of scope for Trilium. But could there be another method to achieve the same? Here's the first thing I though of:
> 
>     Have Trilium (server in hosted case, desktop for users not using server) able to take Gmail credentials and the name of a label.
>     Gmail labels in IMAP look like a folder.
>     Trilium to take any unread message in this "folder" and copy its title and contents into a new note under the note labeled #emailInbox, then mark the email as read.
>     User can set a gmail filter - any email to [myemail+trilium@gmail.com](mailto:myemail+trilium@gmail.com) to skip inbox and go to label.
> 
> And walla! I simply add [myemail+trilium@gmail.com](mailto:myemail+trilium@gmail.com) to my email address book as "Trilium", and now I can forward emails, send from my phone, whatever to that address. It all flows into Trilium and bonus, I get a history in my gmail label too.

**Comment by andjar** [comment]
See this idea about using Tasker for Android to interact with the trilium-sender app: https://github.com/zadam/trilium-sender/issues/26

**Comment by pano9000** [comment]
I like the idea, I use a similar approach, but currently it's a fully manual process on my end.


@eliandoran woul you see that as something for "core" TriliumNext or would you also rather see that as possible plugin (eventually?)

the required npm package to achieve this via IMAP is relatively small (when we compare it to other stuff we ship in core currently): 
https://www.npmjs.com/package/imapflow ~ 700 KB unpacked

**Comment by eliandoran** [comment]
@pano9000 , I would say to implement it as a plugin (maybe even a core plugin that is developed by us), since not everybody would benefit from it.

At some point we'll extract some of the existing note types into core plugins in an effort to reduce the final size.
